react/game_progress.js

```diff
@@ -49,13 +49,13 @@ class GameProgress extends Component {
               </Row>
             }
             <Row style={{marginLeft: "0", marginRight: "0"}} className= "justify-content-md-center" >
-              {this.props.gameTick ? Math.floor(((this.props.gameTick % 3600) / 60)) : '0'}:{this.props.gameTick % 60 < 10 ? 0 : ''}{this.props.gameTick ? this.props.gameTick % 60 : '0'}
+              <div class="display-time clock">{this.props.gameTick ? Math.floor(((this.props.gameTick % 3600) / 60)) : '0'}:{this.props.gameTick % 60 < 10 ? 0 : ''}{this.props.gameTick ? this.props.gameTick % 60 : '0'}</div>
             </Row>
             {facilID && this.state.seatsFullError &&
               <Row style={{marginLeft: "0", marginRight: "0"}} className= "justify-content-md-center">Error: Seats are not Full yet</Row>
             }
         </div>
-        <div style={{width: "35%", top: "15%", left: "50%", marginLeft: "-17.5%", position: "absolute", overflow: "auto"}}>
+        <div style={{width: "900px", top: "10px", left: "50%", transform: "translate(-50%)", position: "relative", overflow: "auto"}}>
```

Comment:
We were trying to avoid using fixed sizes to keep the design responsive. I'm not opposed to using fixed sizes but want to make sure we are doing it intentionally.

Reply:
@ChrisSchreiber I'm not a big fan using fixed sizes here but with the dependency on background and other svg element to be relative to this, I would have to change a lot of CSS styling. I have it on my list though, this is just a quick and easy fix for now to display content within the frame/div